core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
I think the pull of the `MessageDestination` is already too much overhead for sending a message. If you don't agree I can revert the change and implement a two-way relation. I'm not going to combine caching of all messages in this change set because it is altogether uncertain if we are going to do that.